src/ipc/get-battery-info.ts

```diff
@@ -0,0 +1,47 @@
+import { IpcMain } from 'electron'
+import * as fs from 'fs'
+import { promisify } from 'util'
+
+const readFile = promisify(fs.readFile)
+
+export enum BatteryStatus {
+  Charging = 'Charging',
+  Discharging = 'Discharging',
+  NotCharging = 'Not charging',
+  Full = 'Full',
+  Unknown = 'Unknown',
+}
+
+export interface BatteryInfo {
+  status: BatteryStatus
+  level: number
+}
```

Comment:
@beausmith how do you feel about this? I’m basically just piping the status label directly through. Do you want a boolean property that indicates whether its plugged in or not? Otherwise you’d look for “Charging” or “Full”.

Reply:
A boolean is all the BMD needs.

I can see how simply passing the data through keeps the `kiosk-browser` simple. 

I don't foresee any features taking advantage of any finer-grain status though, so I'm tempted to want this to simply return a boolean. 

Alternately, how about just adding a `isCharging` boolean?

```ts
const isCharging = status === 'Charging' || status === 'Full'
```

Reply:
Removed `status` and added `discharging`.